src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -31,59 +34,82 @@ public class SnapshotBuilderTestData {
       new SnapshotBuilderSettings()
           .domainOptions(
               List.of(
-                  new SnapshotBuilderDomainOption()
-                      .id(10)
-                      .category("Condition")
-                      .root(
-                          new SnapshotBuilderConcept()
-                              .id(100)
-                              .name("Condition")
-                              .count(100)
-                              .hasChildren(true)),
-                  new SnapshotBuilderDomainOption()
-                      .id(11)
-                      .category("Procedure")
-                      .root(
-                          new SnapshotBuilderConcept()
-                              .id(200)
-                              .name("Procedure")
-                              .count(100)
-                              .hasChildren(true)),
-                  new SnapshotBuilderDomainOption()
-                      .id(12)
-                      .category("Observation")
-                      .root(
-                          new SnapshotBuilderConcept()
-                              .id(300)
-                              .name("Observation")
-                              .count(100)
-                              .hasChildren(true))))
+                  (SnapshotBuilderDomainOption)
+                      new SnapshotBuilderDomainOption()
+                          .root(
+                              new SnapshotBuilderConcept()
+                                  .id(100)
+                                  .name("Condition")
+                                  .count(100)
+                                  .hasChildren(true))
+                          .tableName("condition_occurrence")
+                          .columnName("condition_concept_id")
```

Comment:
is there a reason only this domain option has tableName and columnName populated? Or do the others just not have it because it isn't required

Reply:
Oh, they should have it. Will add